modin/pandas/test/test_general.py

```diff
@@ -179,6 +180,21 @@ def test_merge():
         pd.merge("Non-valid type", modin_df2)
 
 
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+def test_merge_pandas(data):
+    modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
```

Comment:
this has to be imported from `modin.pandas.test.utils`
```suggestion
    modin_df, pandas_df = create_test_dfs(data)
```